src/main/scala/mesosphere/marathon/core/event/Events.scala

```diff
@@ -9,9 +10,20 @@ import mesosphere.marathon.core.instance.Instance
 import mesosphere.marathon.state.{ AppDefinition, PathId, Timestamp }
 import mesosphere.marathon.upgrade.{ DeploymentPlan, DeploymentStep }
 import org.apache.mesos.{ Protos => Mesos }
+import play.api.libs.json.Json
 
 import scala.collection.immutable.Seq
 
+trait Event {
+  val eventType: String
+  val message: String
+}
+
+case class JsonEvent(event: MarathonEvent) extends Event {
+  val eventType: String = event.eventType
+  lazy val message: String = Json.stringify(eventToJson(event))
+}
+
 sealed trait MarathonEvent {
   val eventType: String
   val timestamp: String
```

Comment:
I'm sorry I don't understand. If you are referring to put `lazy val jsonString` into `MarathonEvent` Then we need to ignore this filed during serialization. I created new class to distinct data from their view. We use JSON now but we can use Protobuf or something else in the future. Should I move `JSONEvent` to SSE Actor?